test/e2e/test_create_and_xmlupload.py

```diff
@@ -0,0 +1,129 @@
+import urllib.parse
+from pathlib import Path
+from typing import Any
+from typing import Iterator
+
+import pytest
+import requests
+
+from dsp_tools.cli.args import ServerCredentials
+from dsp_tools.commands.project.create.project_create import create_project
+from dsp_tools.commands.xmlupload.xmlupload import xmlupload
+from test.e2e.setup_testcontainers import get_containers
+
+PROJECT_SHORTCODE = "4125"
+ONTO_NAME = "e2e-testonto"
+CREDS = ServerCredentials("root@example.com", "test", "http://0.0.0.0:3333")
+ONTO_IRI = f"{CREDS.server}/ontology/{PROJECT_SHORTCODE}/{ONTO_NAME}/v2"
+PROPS_IN_ONTO_JSON = 1
+RESCLASSES_IN_ONTO_JSON = 2
+
+
+@pytest.fixture()
+def _create_project() -> Iterator[None]:
+    with get_containers():
+        success = create_project(Path("testdata/json-project/test-project-e2e.json"), CREDS, verbose=True)
+        assert success
```

Comment:
to me it feels a bit strange to have assertions in fixtures

Reply:
That's indeed particular. For me, it is a way to test our `success` variables that we pass around in the code. A very superficial black-box test, but still.